hack/convert.go

```diff
@@ -0,0 +1,180 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/solo-io/go-utils/log"
+	"github.com/solo-io/skv2/codegen/model"
+	"github.com/solo-io/skv2/codegen/util"
+	"github.com/solo-io/solo-apis/codegen"
+)
+
+func main() {
+	var resources []string
+	for _, group := range codegen.GlooGroups() {
+		for _, resource := range group.Resources {
+			resources = append(resources, resource.Kind)
+		}
+	}
+	moduleRoot := util.GetModuleRoot()
+	err := filepath.Walk(
+		filepath.Join(moduleRoot, "api/gloo"),
+		func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+			if info.IsDir() {
+				return nil
+			}
+			if !strings.HasSuffix(info.Name(), ".proto") {
+				return nil
+			}
+			file, err := os.OpenFile(path, os.O_RDWR, os.ModePerm)
+			if err != nil {
+				return err
+			}
+			defer file.Close()
+			wholeFileBytes, err := ioutil.ReadAll(file)
+			if err != nil {
+				return err
+			}
+			// If the proto file is not from a relevant package skip it.
+			wholeFileBytes, ok := isRelevantFile(wholeFileBytes)
```

Comment:
See the other comment about the misleading name. I think this function can be self-documenting if we split its behavior into meaningful functions, so that it reads:

```
if isRelevantFile(...) -> continue

for relevantTypes {
  1. patch spec message
  2. append status message
  3 add import
  ... 
}

filter out undesired lines

save file
```

Reply:
The isRelevant check is actually a bit - it's 2 lines by itself, so I've changed 'isRelevantFile' to 'modifyRelevantFile'.